src/components/forms/SmallGrantsForm.tsx

```diff
@@ -537,10 +738,29 @@ export const SmallGrantsForm: FC = () => {
                 color='brand.paragraph'
                 fontSize='input'
                 mt={3}
+                {...register('projectRequestedAmount', {
+                  required: isAProject,
+                  maxLength: 20
+                })}
```

Comment:
Wondering here if you have considered using a `number` input field here since you can enter a text value and I think we are not validating that.